core/app/services/workarea/copy_page.rb

```diff
@@ -0,0 +1,31 @@
+module Workarea
+  class CopyPage
+    def initialize(page, attrs = {})
+      @page = page
+      @attributes = Workarea.config.page_copy_default_attributes.merge(attrs)
+    end
+
+    def perform
+      page_copy = @page.clone
+      page_copy.assign_attributes(@attributes)
```

Comment:
```suggestion
      page_copy.attributes = @attributes
```

Reply:
I used `CopyProduct` as my base for this. Copyparsta.